pkg/kubelet/eviction/eviction_manager_test.go

```diff
@@ -117,6 +117,32 @@ func makePodWithDiskStats(name string, priority int32, requests v1.ResourceList,
 	return pod, podStats
 }
 
+func makePodWithLocalStorageCapacityIsolationOpen(name string, priority int32, requests v1.ResourceList, limits v1.ResourceList, memoryWorkingSet string) (*v1.Pod, statsapi.PodStats) {
```

Comment:
I would rather use an enum for name.

Reply:
I follow the pattern from https://github.com/kubernetes/kubernetes/blob/1494839b32aac4104e105bf02bfc6e477a798f06/pkg/kubelet/eviction/eviction_manager_test.go#L104

and
https://github.com/kubernetes/kubernetes/blob/1494839b32aac4104e105bf02bfc6e477a798f06/pkg/kubelet/eviction/eviction_manager_test.go#L112

Reply:
in my case I just use the eviction case name as pod name to match different podStats. So it seems like force code paths.

Reply:
we can use enum as well, just need to add a field on struct ``podToMake``

Reply:
I see.  Maybe we can leave it for now and see what a sig-node reviewer says.